src/generators/static-theme.ts

```diff
@@ -176,57 +176,45 @@ const ruleGenerators = [
     createRuleGen((t) => t.invert, () => ['filter: invert(100%) hue-rotate(180deg)']),
 ];
 
-const staticThemeCommands = [
-    'NO COMMON',
-
-    'NEUTRAL BG',
-    'NEUTRAL BG ACTIVE',
-    'NEUTRAL TEXT',
-    'NEUTRAL TEXT ACTIVE',
-    'NEUTRAL BORDER',
-
-    'RED BG',
-    'RED BG ACTIVE',
-    'RED TEXT',
-    'RED TEXT ACTIVE',
-    'RED BORDER',
-
-    'GREEN BG',
-    'GREEN BG ACTIVE',
-    'GREEN TEXT',
-    'GREEN TEXT ACTIVE',
-    'GREEN BORDER',
-
-    'BLUE BG',
-    'BLUE BG ACTIVE',
-    'BLUE TEXT',
-    'BLUE TEXT ACTIVE',
-    'BLUE BORDER',
-
-    'FADE BG',
-    'FADE TEXT',
-    'TRANSPARENT BG',
-
-    'NO IMAGE',
-    'INVERT',
-];
-
-function upperCaseToCamelCase(text: string) {
-    return text
-        .split(' ')
-        .map((word, i) => {
-            return (i === 0
-                ? word.toLowerCase()
-                : (word.charAt(0).toUpperCase() + word.substr(1).toLowerCase())
-            );
-        })
-        .join('');
-}
+const staticThemeCommands = {
+    'NO COMMON': 'noCommon',
+
+    'NEUTRAL BG': 'neutralBg',
+    'NEUTRAL BG ACTIVE': 'neutralBgActive',
+    'NEUTRAL TEXT': 'neutralText',
+    'NEUTRAL TEXT ACTIVE': 'neutralTextActive',
+    'NEUTRAL BORDER': 'neutralBorder',
+
+    'RED BG': 'redBg',
+    'RED BG ACTIVE': 'redBgActive',
+    'RED TEXT': 'redText',
+    'RED TEXT ACTIVE': 'redTextActive',
+    'RED BORDER': 'redBorder',
+
+    'GREEN BG': 'greenBg',
+    'GREEN BG ACTIVE': 'greenBgActive',
+    'GREEN TEXT': 'greenText',
+    'GREEN TEXT ACTIVE': 'greenTextActive',
+    'GREEN BORDER': 'greenBorder',
+
+    'BLUE BG': 'blueBg',
+    'BLUE BG ACTIVE': 'blueBgActive',
+    'BLUE TEXT': 'blueText',
+    'BLUE TEXT ACTIVE': 'blueTextActive',
+    'BLUE BORDER': 'blueBorder',
+
+    'FADE BG': 'fadeBg',
+    'FADE TEXT': 'fadeText',
+    'TRANSPARENT BG': 'transparentBg',
+
+    'NO IMAGE': 'noImage',
+    'INVERT': 'invert',
+};
 
 export function parseStaticThemes($themes: string) {
     return parseSitesFixesConfig<StaticTheme>($themes, {
-        commands: staticThemeCommands,
-        getCommandPropName: upperCaseToCamelCase,
+        commands: Object.keys(staticThemeCommands),
+        getCommandPropName: (command) => staticThemeCommands[command] || null,
```

Comment:
```suggestion
        getCommandPropName: (command) => staticThemeCommands[command],
```

It will by default return `undefined` or the value, no need to fallback to `null`.

Reply:
I agree that `|| null` is unnecessary. Should I also remove it from these two places?
https://github.com/darkreader/darkreader/blob/8a56257b84f8a2d7adef27d1018b5f3ac383943f/src/generators/dynamic-theme.ts#L17
https://github.com/darkreader/darkreader/blob/8a56257b84f8a2d7adef27d1018b5f3ac383943f/src/generators/css-filter.ts#L215

Reply:
Fine by me.

Reply:
Done!